### PR TITLE
fix(ci): Do not initialize image trust store on every convey

### PR DESCRIPTION
**Pull request by andaaron**
Should miticage failures such as https://github.com/project-zot/zot/actions/runs/6148316168/job/16681757801?pr=1788

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: pkg/meta/dynamodb/dynamodb_test.go [hunk 1]
```diff
@@ -151,6 +151,11 @@ func TestWrapperErrors(t *testing.T) {
 
 	log := log.NewLogger("debug", "")
 
+	imgTrustStore, err := imagetrust.NewAWSImageTrustStore(region, endpoint)
+	if err != nil {
+		panic(err)
```
Comment (rchincha) [inline pkg/meta/dynamodb/dynamodb_test.go:156]: We have a logger here right? Can we log.Fatal() instead?

Comment (andaaron) [reply pkg/meta/dynamodb/dynamodb_test.go:156]: This is a test, it needs to exit if it cannot create the object.

Comment (andaaron) [reply pkg/meta/dynamodb/dynamodb_test.go]: Switch to using the test framework specific methods.

**Comment by andaaron** [comment]
Closing this as nobody has experienced that failure since Andreea's fix.

**Pull request closed without merging**